x/mongo/driver/topology/pool.go

```diff
@@ -767,6 +767,80 @@ func (p *pool) removeConnection(conn *connection, reason reason, err error) erro
 	return nil
 }
 
+var (
+	// BGReadTimeout is the maximum amount of the to wait when trying to read
+	// the server reply on a connection after an operation timed out. The
+	// default is 400ms.
+	//
+	// Deprecated: BGReadTimeout is intended for internal use only and may be
+	// removed or modified at any time.
+	BGReadTimeout = 400 * time.Millisecond
```

Comment:
Use a lower timeout value here compared to v1.15 because users have observed increased database load caused by this feature in some circumstances. Reducing the timeout value reduces the amount of time that server-side operations will continue after a client-side timeout, mitigating the increased database load.